### DISCUSSION TITLE
https://klp1.com/parts/toolhead-parts/

### DISCUSSION THREAD

**Discussion by giscus** [blakadder/klp1#3] [Announcements]
# https://klp1.com/parts/toolhead-parts/

Toolhead parts (extruder and hotend) for Kingroon KLP1

https://klp1.com/parts/toolhead-parts/

**Comment by grvtta** [comment]
The thermistor from aliexpress is completely wrong, it has a different port and also a different head

**Comment by iRaiFx** [comment]
The extruder PTFE quick connector is PC4-M6 if anyone wondering.

**Comment by phb98** [comment]
the nozzle is not e3d v6, the e3d v6 has 7.5mm thread length, and kingroon nozzle is around 8mm, so using e3d nozzle will lead to leakage. Learn it the hard way

**Reply by wFirbolg** [reply]
That's exactly what happened to me as well! Even with V6 nozzles bought from the website (E3D V6 CHT Hardened Steel Nozzle High Flow Nozzle), I'm getting that same issue. Could I change the heatbrake for something else?

**Comment by wFirbolg** [comment]
I think my probe stopped working and now my bed hits the nozzle when I home the axis. I have the KLP1-230 printer and it uses a similar probe to Sovol's SV08 (https://www.3djake.ch/en-CH/sovol/auto-levelling-sensor-1) so, should I get I ask or buy a new probe from Kingroon (my specific probe is not available on the website)? Get the sovol one? Or am I missing something. I started to happen when I cleanup a clog and reassembled the toolhead but was working fine at first and then started to hit the table.  Thanks

**Comment by asgard439** [comment]
What is the GPIO number on the new toolhead board in the connector with the marking MTDET?
